### PR TITLE
Story 5

### PR DESCRIPTION
**Pull request by limsammy**
Related Issue: #23 

Summary: User can now click "Remove Item" to remove an item from their cart. Other items will still persist.

Author(s): sam

### CODE REVIEW

File: app/controllers/carts_controller.rb [hunk 1]
```diff
@@ -1,8 +1,7 @@
 class CartsController < ApplicationController
 
   def show
-    @items = Item.where(id: session[:cart].values)
-    # @cart =
```
Comment (randy-darsh) [inline app/controllers/carts_controller.rb:5]: im stoked that you deleted the commented out line

File: app/controllers/carts_controller.rb [hunk 2]
```diff
@@ -1,8 +1,7 @@
 class CartsController < ApplicationController
 
   def show
-    @items = Item.where(id: session[:cart].values)
-    # @cart = 
+    @items = Item.where(id: session[:cart].keys)
```
Comment (amhursh) [inline app/controllers/carts_controller.rb:4]: Good catch. Should work moving forward.

File: app/controllers/carts_controller.rb [hunk 3]
```diff
@@ -12,4 +11,9 @@ def create
     session[:cart][id] = (session[:cart][id] || 0) + 1
     redirect_back(fallback_location: items_path)
   end
+
+  def destroy
+    session[:cart].delete(params[:id].to_s)
```
Comment (amhursh) [inline app/controllers/carts_controller.rb:16]: excellent. Interesting way to clear the cart item. Think it should work well.

File: app/views/carts/show.html.erb [hunk 4]
```diff
@@ -3,6 +3,7 @@
 <h5><%= item.title %></h5>
 <p><%= item.description %></p>
 <p>$<%= item.price %></p>
+<%= link_to "Remove Item", cart_path(id: item.id), method: :delete %>
```
Comment (randy-darsh) [inline app/views/carts/show.html.erb:6]: good catch moving it

**Pull request merged**